src/mouse-keyboard-vr-display.js

```diff
@@ -161,6 +164,28 @@ MouseKeyboardVRDisplay.prototype.onMouseUp_ = function(e) {
   this.isDragging_ = false;
 };
 
+
+MouseKeyboardVRDisplay.prototype.requestPointerLock_ = function(element) {
+  // from pointerlock controls in three.js
+  // - https://github.com/mrdoob/three.js/blob/master/examples/misc_controls_pointerlock.html
+  element.requestPointerLock = element.requestPointerLock || element.mozRequestPointerLock || element.webkitRequestPointerLock;
+  if ( /Firefox/i.test( navigator.userAgent ) ) {
+    var fullscreenchange = function ( event ) {
+      if ( document.fullscreenElement === element || document.mozFullScreenElement === element ) {
+        document.removeEventListener( 'fullscreenchange', fullscreenchange );
```

Comment:
i will try to have find a IE, but their docs says pointerlockrequest is supported. i hope at least :) 
https://msdn.microsoft.com/en-us/library/mt560346(v=vs.85).aspx